### ISSUE TITLE
Fails if extension lives in subdirectory

### ISSUE THREAD

**Issue by Treora** [mmktomato/parcel-plugin-web-ext-tool#8]
The example setup has `manifest.json` in the same directory as `package.json`, so `web-ext` finds it automatically. However, if the manifest is moved into a subdirectory `extension` (and parcel’s output is directed there using `-d extension/dist`), one would have to supply the `sourceDir` option to `web-ext` to ensure it finds it. I tried two approaches:

To try make this work, I added a line to package.json: `"webExt": { "sourceDir": "extension" }`. This makes the extension work when run directly using `npx web-ext run` (after running `parcel build`). But this option seems to be ignored by `parcel-plugin-web-ext-tool`, which only appears to load the config in `webExt.run`.

I therefore tried moving the option there: `"webExt": { "run": { "sourceDir": "extension" } }`. But this both appears to be invalid when invoking web-ext directly (`unknown option: "sourceDir"`), and using `parcel-plugin-web-ext-tool` still fails:

> … installTemporaryAddon: Error: unknownError: Could not install add-on at './extension': [Exception... "Component returned failure code: 0x80520001 (NS_ERROR_FILE_UNRECOGNIZED_PATH) [nsIFile.initWithPath]" …

Any idea why this happens, or how to make it work when the extension lives in a subdirectory?

**Comment by Treora** [comment]
Forgot to mention my setup:

- web-ext 5.4.0
- parcel 0.2.0 (current master)
- Firefox 76, on Linux

I could also imagine that a bug in web-ext causes this ‘unrecognised path’ error. Besides that, I don’t see a reason why the custom configuration for `run` could not define a `sourceDir`.

**Comment by mmktomato** [comment]
Let me confirm. Is this what you say?
```
(project root)
|-- package.json
|-- extension
    |-- manifest.json
    |-- (your source files are here)
```
And you run `parcel watch` in `(project root)`?

**Comment by Treora** [comment]
Roughly like that indeed; more precisely:

```
(project root)
|-- package.json
|-- src
    |-- content.js
|-- extension
    |-- manifest.json
    |-- dist
        |-- content.js
```

> And you run `parcel watch` in `(project root)`?

Yes. In `package.json`: `"start": "parcel watch src/content.js -d extension/dist --public-url=/dist"`

Here is this minimal example using the example setup contained within this repo: <https://github.com/Treora/parcel-plugin-web-ext-tool/commit/50f179c81123eb958a9d70361709ad78700edf45>

**Comment by mmktomato** [comment]
Thank you. I'll check it.

**Comment by mmktomato** [comment]
@Treora I published `v0.2.1` including a fix for this issue. Could you try it?

**Comment by Treora** [comment]
> @Treora I published `v0.2.1` including a fix for this issue. Could you try it?

Sorry I totally forgot about this! I just checked, it works. Thanks for fixing! :)
